### PR TITLE
Update logic filter on trade history page

### LINKED ISSUE #341: fix filtering and searching for the trade history page
1. add "All Severs" and "All" as part of the server dropdown and status dropdown respectively

Image

2.  fix the filtering for the newest/oldest dropdown, and fix filtering for the server drodpdown

3. fix the search bar

### PR DESCRIPTION
**Pull request by JustinBeBoy**
Resolve #341

### CODE REVIEW

File: libwallet/instantswap/instantswap.go [hunk 1]
```diff
@@ -127,18 +128,28 @@ func (instantSwap *InstantSwap) NewExchangeServer(exchangeServer ExchangeServer)
 // GetOrdersRaw fetches and returns all saved orders.
 // If status is specified, only orders with that status will be returned.
 // status is made optional to the sync functionality can update all orders.
-func (instantSwap *InstantSwap) GetOrdersRaw(offset, limit int32, newestFirst bool, status ...instantswap.Status) ([]*Order, error) {
+func (instantSwap *InstantSwap) GetOrdersRaw(offset, limit int32, newestFirst bool, server, txID string, status ...instantswap.Status) ([]*Order, error) {
 	var query storm.Query
```
Comment (ukane-philemon) [inline libwallet/instantswap/instantswap.go:132]: rm. Can just assign this at L151,

File: ui/page/exchange/order_history_page.go [hunk 2]
```diff
@@ -173,7 +177,10 @@ func (pg *OrderHistoryPage) HandleUserInteractions() {
 func (pg *OrderHistoryPage) initServerSelector() {
 	pg.exchangeServers = pg.AssetsManager.InstantSwap.ExchangeServers()
 
-	items := []cryptomaterial.DropDownItem{}
+	items := []cryptomaterial.DropDownItem{{
+		// Icon: pg.Theme.Icons.WalletIcon,
```
Comment (ukane-philemon) [inline ui/page/exchange/order_history_page.go:181]: rm?

**Review by ukane-philemon — APPROVED** [review]
UTAck. Looks good, just a few nits.

**Review by dreacot — APPROVED** [review]
Ack

**Pull request merged**